yaml_decode/mono_pkg.yaml

```diff
@@ -0,0 +1,10 @@
+# See https://github.com/dart-lang/mono_repo for details
+stages:
+- analyzer_and_format:
+  - group:
+    - dartfmt
+    - dartanalyzer: --fatal-warnings --fatal-infos .
+    dart: [dev]
```

Comment:
nit: indent these to line up with the command

Reply:
Explain?

Reply:
oh wait, these are for the `group` - I thought this was for the `-dartanalyzer:` task. The indentation here is really confusing. I guess my suggestions then would be to intend the list items instead so they aren't at the same level as the map keys.

Reply:
```yaml
  - group:
      - dartfmt
      - dartanalyzer: --fatal-warnings --fatal-infos .
    dart: [dev]
```

Reply:
this is actually the yaml formatting my IDE does. Not that I'm used to it, I kinda like it...

But I can back out...

Reply:
is it configurable in whatever plugin you are using?

Reply:
Sadly, no....